Monsters hunting persistence/src/main/java/cz/fi/muni/pa165/dao/AreaDaoImpl.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cz.fi.muni.pa165.dao;
+
+import cz.fi.muni.pa165.entity.Area;
+import cz.fi.muni.pa165.enums.AreaType;
+import java.util.List;
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.PersistenceContext;
+import static org.assertj.core.util.Preconditions.checkNotNull;
+
+/**
+ *
+ * @author Jan Gol <gol.honza@gmail.com>
+ */
+public class AreaDaoImpl implements AreaDao {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Override
+    public void create(Area area) {
+        checkNotNull(area);
```

Comment:
Do not use this method in implementation. It is ment to be used in tests. Actually, maven should add this library only in tests so for real production the project wont run at all. Write this the same way as you have in `findByName` method.